test/betterC/typecons.d

```diff
@@ -0,0 +1,7 @@
+extern(C) void main()
```

Comment:
Can you now move it inside `std.typecons` as a `@BetterC unittest`?

Reply:
I can/will do this once https://github.com/dlang/phobos/pull/6645 has been merged.